src/dev/lupio/LupioRTC.py

```diff
@@ -0,0 +1,19 @@
+from m5.params import *
```

Comment:
This file needs a license text. Copyright should be UC Regents. See src/arch/riscv/pmp.cc for an example.